src/components/ReactSlider/ReactSlider.jsx

```diff
@@ -222,25 +222,26 @@ class ReactSlider extends React.Component {
 
         /**
          * Callback called before starting to move a thumb. The callback will only be called if the
-         * action will result in a change. The function will be called with one argument,
-         * the initial value(s).
+         * action will result in a change. The function will be called with two arguments, the first
+         * being the new value(s) the second being thumb index.
          */
         // eslint-disable-next-line max-len
         // eslint-disable-next-line zillow/react/require-default-props, zillow/react/no-unused-prop-types
         onBeforeChange: PropTypes.func,
 
         /**
          * Callback called on every value change.
-         * The function will be called with one argument, the new value(s).
+         * The function will be called with two arguments, the first being the new value(s) 
+         * the second being thumb index.
          */
         // eslint-disable-next-line max-len
         // eslint-disable-next-line zillow/react/require-default-props, zillow/react/no-unused-prop-types
         onChange: PropTypes.func,
 
         /**
          * Callback called only after moving a thumb has ended. The callback will only be called if
-         * the action resulted in a change. The function will be called with one argument,
-         * the result value(s).
+         * the action resulted in a change. The function will be called with two arguments, the
+         * first being the new value(s) the second being thumb index.
```

Comment:
Can you change "new value(s)" back to "result value(s)"?

Reply:
done